WordPress/src/main/java/org/wordpress/android/ui/prefs/DeleteSiteDialogFragment.java

```diff
@@ -76,7 +83,7 @@ public void onClick(DialogInterface dialog, int which) {
             public void onClick(DialogInterface dialog, int which) {
                 Fragment target = getTargetFragment();
                 if (target != null) {
-                    target.onActivityResult(getTargetRequestCode(), Activity.RESULT_OK, null);
+                    target.onActivityResult(getTargetRequestCode(), Activity.RESULT_OK, new Intent());
```

Comment:
It was tricky, but I think you're right. I had a fear of readability in `onActivityResult`, but it looks fine even with the multiple branches. Addressed in c1e4f86dfa3ea4990cf020df21e2e11535e1fef4